docs/pages/custom-data.rst

```diff
@@ -80,6 +80,30 @@ Python's `inspect.getargspec` is used to only pass the arguments declared by the
 function. This means it's not necessary to add a catch all (``**``) keyword
 argument.
 
+The render function can also be used to display the data from two columns in one column. In the following example we can see the field `last_name` is appended to the `name` field using the render function. Note that `value` is return the value in the column and `record` is used to access the values in the `last_name` column::
+
+    #models.py
```

Comment:
Please try to use coding style consistend with the rest of the code base (i.e., it should pass flake8). In this case, a space after the hash-char

Reply:
Sorry about that I just issues with spacing because I disabled PEP-8  on the IDE. I will see if I can put in pyflakes or something.